queries/gitcommit/highlights.scm

```diff
@@ -9,6 +9,7 @@
 
 (subject) @text.title @spell
 (subject (overflow) @text @spell)
+(subject (subject_prefix) @function @spell)
```

Comment:
Does it really need `@spell` when it's that limited (only `amend!` or `fixup!`)

Reply:
Oh! you're right, it's a mistake, I wanted to add `@nospell` :sweat_smile: